examples/queue-manager-demo/package.json

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
You can use `PendingSwapNetworkStatus` from `rango-types` in `examples/queue-manager-demo/src/flows/rango/types.ts`

Reply:
I tried to update it but it seems that almost all files in this package need to be updated. I think we could create a task for updating queue-manager-demo project if it's still used.